### PR TITLE
Update humans.txt to mark Ben Thorner's departure

### PR DESCRIPTION
**Pull request by kevindew**
Today is Ben Thorner's last day at GDS. As per the documentation he very helpfully wrote [1] we'd like to make a special gesture to thank him for his exceptional positive impact to GDS.

[1]: https://github.com/alphagov/static/blob/577fd669535a8f5e44e26c2764e54a63ff51b793/docs/humans.md

### CODE REVIEW

File: public/humans.txt [hunk 1]
```diff
@@ -1,2 +1,10 @@
 GOV.UK is built by a team at the Government Digital Service (with thanks to Martha!) in London,
-Bristol and Manchester.  If you'd like to join us, see https://gdscareers.gov.uk/
\ No newline at end of file
+Bristol and Manchester.  If you'd like to join us, see https://gdscareers.gov.uk/
+
+Today we say goodbye to Ben Thorner. Ben has worked in GDS as a software engineer since 2018 and has made a substantial impact improving GOV.UK’s tooling and development practices.
+
+We have many contributions to thank Ben for. He has helped deliver key government priorities. He has created communities to help improve developer experience and tooling. He has tirelessly facilitated opinionated discussions with many stakeholders to lead to fruitful results. He’s created and iterated tooling that all GOV.UK software developers benefit from each day.
```
Comment (andyhd) [inline public/humans.txt:6]: I think "He's created" should be "He has created"

File: public/humans.txt [hunk 2]
```diff
@@ -1,2 +1,10 @@
 GOV.UK is built by a team at the Government Digital Service (with thanks to Martha!) in London,
-Bristol and Manchester.  If you'd like to join us, see https://gdscareers.gov.uk/
\ No newline at end of file
+Bristol and Manchester.  If you'd like to join us, see https://gdscareers.gov.uk/
+
+Today we say goodbye to Ben Thorner. Ben has worked in GDS as a software engineer since 2018 and has made a substantial impact improving GOV.UK’s tooling and development practices.
+
+We have many contributions to thank Ben for. He has helped deliver key government priorities. He has created communities to help improve developer experience and tooling. He has tirelessly facilitated opinionated discussions with many stakeholders to lead to fruitful results. He’s created and iterated tooling that all GOV.UK software developers benefit from each day. 
+
+However, we’re most grateful for his personal contributions to our community: his ability to share information he’s learnt, his willingness to collaborate and his unwavering enthusiasm that things can be better and it's within our reach to make them so.
```
Comment (andyhd) [inline public/humans.txt:8]: and "he's learnt" should be "he has learnt"

Comment (andysellick) [inline public/humans.txt:8]: Apostrophes are inconsistent? `'` and `’`

**Review by andysellick — APPROVED** [review]
😢

**Review by andyhd — APPROVED** [review]
I don't know why I dislike the "he has (done something)" -> "he's" contraction and not the others ("we're", "it's") - feel free to ignore me!

**Comment by kevindew** [comment]
Thanks both. I've made amends. Will merge when it's green.

**Review by richardTowers — APPROVED** [review]
:100:

**Pull request merged**